scripts/agd-builder.sh

```diff
@@ -82,11 +82,14 @@ fi
 if $need_nodejs; then
   # We need to get node at the script top level because it's used by the daemon
   # as well.
-  case $(node --version 2> /dev/null) in
+  found_node_version=$(node --version 2> /dev/null)
+  case "$found_node_version" in
     "$NODEJS_VERSION" | "$NODEJS_VERSION".*) ;;
+    "$NODEJS_ALT_VERSION" | "$NODEJS_ALT_VERSION".*) ;;
     *)
-      # Auto-download the NodeJS version we need, if allowed.
-      $SKIP_DOWNLOAD || {
+      $SKIP_DOWNLOAD && fatal "Unsupported version of Node.js $found_node_version. Use either $NODEJS_VERSION or $NODEJS_ALT_VERSION"
```

Comment:
I guess this matches the `expr || Fail` assertion style but consider an actual if/then

Reply:
Reverted